test/mocks/values/category.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  CATEGORY: 'correlationId'
```

Comment:
correlationId is a category? This might be clearer when I come to review the tests maybe...

Reply:
Yes, when saving to the event store we give each row a category so we know what the partition key is.  This repo only cares about those where the partition key is a correlation id, so that's what we've set in the mock.